### DISCUSSION TITLE
Did anyone upgrade to HASS OS 10 to find out this awesome integration stopped working?

### DISCUSSION THREAD

**Discussion by Zoriontsu** [cytech/Home-Assistant-wundergroundpws#167] [Q&A]
I have another instance still running on 9.5 and it is working fine.

![image](https://user-images.githubusercontent.com/22824068/232834012-e6073df3-fbd9-4a66-b6a4-f37e43703c48.png)

![image](https://user-images.githubusercontent.com/22824068/232834358-7aebcc87-d5c4-43af-b306-cc3f54f2b65d.png)

**Comment by nepozs** [comment]
I'm on HAOS 10.2 (and also 10.1 and 10.0 were used) + Wundergroundpws 2.01 without problems.

**Comment by norburban** [comment]
If you're still having issues, try rebooting the hardware HA is running on.

**Comment by cytech** [comment]
closed as outdated
